### PR TITLE
W-16462842 resource redistribution updates

### PR DESCRIPTION
**Pull request by valkyrie69**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/business-groups.adoc [hunk 1]
```diff
@@ -132,20 +133,25 @@ include::partial$include-nav-organizations.adoc[]
 [[redistribute-resources]]
 == Redistribute Resources Between Existing Business Groups
 
-Sometimes you need to redistribute resources (such as vCores, VPCs, load balancers, and so on) based on deployment of apps in your business groups.
-The business group hierarchy prevents a child business group from consuming resources from either its parent group or the root organization unless an Organization Administrator redistributes those resources. Resource distribution proceeds from root to parent to child.
-For example, a child business group cannot receive sandbox vCores if the parent business group does not have any sandbox vCores available, even if the root organization has sandbox vCores available.
-Additionally, the root organization or parent business group cannot reclaim resources that are currently allocated to a business group that is lower in the hierarchy.
+If you need to redistribute resources (such as vCores, VPCs, load balancers, and so on), you can redistribute resources between business groups at the business group level only (not at the root organization level).
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/business-groups.adoc]:
```suggestion
You can redistribute resources (such as vCores, VPCs, load balancers, and so on) between business groups at the business group level only, not at the root organization level. 
```
Minor optional suggestion to reduce wording.

File: modules/ROOT/pages/business-groups.adoc [hunk 2]
```diff
@@ -132,20 +133,25 @@ include::partial$include-nav-organizations.adoc[]
...
-Additionally, the root organization or parent business group cannot reclaim resources that are currently allocated to a business group that is lower in the hierarchy.
+If you need to redistribute resources (such as vCores, VPCs, load balancers, and so on), you can redistribute resources between business groups at the business group level only (not at the root organization level). 
 
-You must have the Organization Administrator permission to redistribute resources.
+The business group hierarchy prevents a child business group from consuming additional resources from its parent group unless an organization administrator redistributes those resources to the child group. Resource distribution proceeds from root to parent to child. For example, you can't redistribute sandbox VCores to a child business group if the parent business group doesn't have any sandbox vCores available, even if the root organization has sandbox vCores available.
 
-include::partial$include-nav-business-groups.adoc[]
+Additionally, the root organization or parent business group can't reclaim resources that are allocated to a business group that isn't a direct child.
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/business-groups.adoc]:
```suggestion
Additionally, the root organization or parent business group can't reclaim resources allocated to a business group that isn't a direct child.
```

**Review by Cristian-Venticinque — DISMISSED** [review]
Approved with minor suggestions

**Review by Cristian-Venticinque — APPROVED** [review]

**Review by sonapetros — APPROVED** [review]
LGTM

**Pull request merged**